### PR TITLE
Strings: rename some vars for clarity purposes

### PR DESCRIPTION
**Pull request by djbe**
Implements #5

Related to https://github.com/SwiftGen/templates/pull/43

### CODE REVIEW

File: Sources/Stencil/StringsContext.swift [hunk 1]
```diff
@@ -67,7 +70,7 @@ extension StringsFileParser {
     )
     let tables: [[String: Any]] = [[
       "name": tableName,
-      "strings": structuredStrings
+      "levels": structuredStrings
```
Comment (AliSoftware) [inline Sources/Stencil/StringsContext.swift:73]: Shouldn't we keep the `strings` entry too (duplicate of `levels`) for 4.2.1 until we can remove it in 5.0.0, at least f we want to merge that for 4.2.1?

**Comment by djbe** [comment]
Lemme check, but I don't think `tables.strings` was ever released.

File: Sources/Stencil/StringsContext.swift (hunk 1 above)
Comment (djbe) [reply Sources/Stencil/StringsContext.swift:73]: Lemme check, but I don't think `tables.strings` was ever released.

Comment (AliSoftware) [reply Sources/Stencil/StringsContext.swift:73]: Ah, good point, maybe not ^^

Comment (djbe) [reply Sources/Stencil/StringsContext.swift:73]: Jup, never released:
https://github.com/SwiftGen/SwiftGenKit/blob/d2760060fa548ca0a8bc78a54cf7d84853494482/Sources/Stencil/StringsContext.swift#L72

(that's SwiftGenKit 1.0.1)

File: Sources/Stencil/StringsContext.swift [hunk 2]
```diff
@@ -16,17 +16,17 @@ private extension String {
...
  - `tables`: `Array` — List of string tables
-   - `name`   : `String` — name of the `.strings` file (usually `"Localizable"`)
-   - `strings`: `Array` — Tree structure of strings (based on dot syntax), each level has:
-     - `name`   : `String` — name of the level (that is, part of the key split by `.` that we're describing)
-     - `strings`: `Array` — list of strings at this level:
-       - `key`: `String` — the full translation key, as it appears in the strings file
+   - `name`  : `String` — name of the `.strings` file (usually `"Localizable"`)
+   - `levels`: `Array` — Tree structure of strings (based on dot syntax), each level has:
+     - `name`    : `String` — name of the level (that is, part of the key split by `.` that we're describing)
+     - `children`: `Array` — list of sub-levels, repeating the structure mentioned above
```
Comment (AliSoftware) [inline Sources/Stencil/StringsContext.swift]: Mentioned below?
Repeating the same structure as a level?

File: Sources/Stencil/StringsContext.swift [hunk 3]
```diff
@@ -37,8 +37,11 @@ extension StringsFileParser {
       let keytail = keyStructure.joined(separator: ".")
```
Comment (AliSoftware) [inline Sources/Stencil/StringsContext.swift]: Maybe rename the variable as well in the code for consistency?

File: Sources/Stencil/StringsContext.swift [hunk 4]
```diff
@@ -135,6 +138,9 @@ extension StringsFileParser {
     }
 
     if !subenums.isEmpty {
```
Comment (AliSoftware) [inline Sources/Stencil/StringsContext.swift]: Maybe rename the variable in the code too for consistency

**Pull request merged**